backend/api/newsletters/mailchimp.py

```diff
@@ -0,0 +1,24 @@
+from dataclasses import dataclass
```

Comment:
this file is inside `api/` which doesn't seem right, in `api` we should only have GraphQL related stuff, this is more of an integration

We already have a `integrations` namespace in the pycon backend source, so we could move this file there?